src/templates/next/components/Button/Button.stories.tsx

```diff
@@ -23,38 +23,88 @@ Default.args = {
   href: "/faq",
 }
 
-export const WithRightIcon = Template.bind({})
-WithRightIcon.args = {
+export const White = Template.bind({})
```

Comment:
can we just do a 2 level map over `colorScheme` + `varaint`

Reply:
don't understand this comment, can elaborate?

Reply:
The current way of doing seems like we'd need to create different variations manually when all that's changing is `colorScheme` and `variant`. 

Maybe we should do a 2 level map like
```ts
variants.map(() => colourSchemes.map)
```

Reply:
[2437d66](https://github.com/isomerpages/isomer-components/pull/22/commits/2437d66a722c85b851a2d420ad60d35b7b796158)

https://65dd949d255950353c94d5cc-efgijxxgyf.chromatic.com/?path=/story/next-components-button--colors-and-variants